getting_started/step_by_step/instancing.rst

```diff
@@ -88,7 +88,7 @@ Click on it and drag it towards the center of the view.
 
 .. image:: img/instancing_ball_moved.png
 
-Play the game by pressing :kbd:`F5` (:kbd:`Cmd + B` on macOS). You should see it fall.
+Play the game by pressing :kbd:`F5` (:kbd:`Cmd + R` on macOS). You should see it fall.
```

Comment:
This is incorrect, the setting is `Cmd + B` according to the settings, have you changed it in yours?

Reply:
You can however change the reference to `F5` below to also include the macOS key

Reply:
You are right. I was probably on the `Script` tab and `Cmd + B` is not working that make me confuse.